frontend/tests/components/admin/report.test.tsx

```
@@ -104,19 +112,314 @@ describe("ReportBlock", () => {
+      // The component checks for clamping, so See More may or may not appear
+      // depending on the mock effectiveness
+    });
+
+    it("expands description when See More is clicked", () => {
+      const longDescReport = {
+        ...mockReport,
+        description: "Long description text that should be clamped initially",
+      };
+
+      // Mock to simulate clamped text
+      const mockScrollHeight = jest
```

ESLint

[error] 'mockScrollHeight' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)